### PR TITLE
Add custom DocumentMetadataComponent to remove dl-invert class

### PR DESCRIPTION
**Pull request by jcoyne**

### CODE REVIEW

File: config/environments/development.rb [hunk 1]
```diff
@@ -69,7 +69,7 @@
   # config.i18n.raise_on_missing_translations = true
 
   # Annotate rendered view with file names.
-  # config.action_view.annotate_rendered_view_with_filenames = true
+  config.action_view.annotate_rendered_view_with_filenames = true
```
Comment (dnoneill) [inline config/environments/development.rb:72]: What does this do?

File: app/controllers/catalog_controller.rb [hunk 2]
```diff
@@ -48,6 +48,7 @@ class CatalogController < ApplicationController
 
     config.show.sidebar_component = Earthworks::Document::SidebarComponent
     config.show.document_component = Earthworks::DocumentComponent
+    config.show.metadata_component = DocumentMetadataComponent
```
Comment (dnoneill) [inline app/controllers/catalog_controller.rb:51]: This should probably be Earthworks::DocumentMetadataComponent

File: config/environments/development.rb (hunk 1 above)
Comment (jcoyne) [reply config/environments/development.rb:72]: It shows you in HTML comments in the development server which views are being rendered.

File: app/controllers/catalog_controller.rb (hunk 2 above)
Comment (jcoyne) [reply app/controllers/catalog_controller.rb:51]: I don't understand why we'd bother with namespaces in an application. That makes sense in a library, but for local stuff it just gets wordy.

Comment (marlo-longley) [reply app/controllers/catalog_controller.rb:51]: I'd be down to change that, that makes sense. We have ~12 components with the namespace right now though. Consistency?

Comment (jcoyne) [reply app/controllers/catalog_controller.rb:51]: https://github.com/sul-dlss/earthworks/pull/1249

**Pull request merged**